style.css

```diff
@@ -0,0 +1,188 @@
+* {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  height: 52px;
+  display: flex;
+  align-items: center;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.24), 0px 0px 4px rgba(0, 0, 0, 0.12);
+}
+
+.header__icon {
+  width: 18px;
+  height: 12px;
+  padding: 0 35px 0 19px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.header__icon-stick {
+  width: 18px;
+  height: 2px;
+
+  background: rgba(0, 0, 0, 0.54);
+}
+
+.header__h1 {
```

Comment:
Please. avoid using tag names in classes. It makes support and refactoring more difficult.